Generator.Application/Models/RequestResult.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Collections.Generic;
+
+namespace Generator.Application.Models
+{
+    public class RequestResult
+    {
+        public RequestResult(List<string> message = null, bool success = true)
```

Comment:
pamietaj, ze zawsze powinienes prosić o jak najmniej concrete type, w tym wypadku nei potrzebujesz cech Listy, wystarcczy Ci IEnumerable. A nawet jesli potrzebowalbys indexux, to zażądaj ICollection albo chociaż IList